projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -399,6 +435,20 @@ public static function block_attributes_to_shortcode_attributes( $atts, $type, $
 
 		// Process inner blocks to shortcode attributes.
 		if ( $block && ! empty( $block->parsed_block['innerBlocks'] ) ) {
+			/*
+			 * Add the `wp-block-jetpack-field-*` and `is-style-*` classes to the field wrapper div
+			 * This ensures any updates to field block styles in theme.json or global styles are
+			 * correctly applied.
+			 */
```

Comment:
I believe prior to #42890, the choice fields used inner blocks for the inner options (just not the label as well). This might result in the `wp-block-jetpack-field-*` class being added for those legacy choice fields. 

This probably isn't a big deal. An alternative might be when processing the inner blocks for the new inner block types, flag the field block as needing the `wp-block` class and updating accordingly based on that.

Reply:
Thanks for flagging. I'll make a note to test next week 🙇🏻

Reply:
Got a draft going... updated tests to come

- https://github.com/Automattic/jetpack/pull/43268